lib/pluginmanager/generate.rb

```diff
@@ -92,6 +92,7 @@ def options
       :author => git_data.author,
       :email  => git_data.email,
       :min_version => "2.0",
+      :logstash_path => File.expand_path('../..', File.dirname(__FILE__))
```

Comment:
There may be a Logstash library variable/function for the root of the project? If so, I'd rather use that than `../..`, but I wasn't aware of one.